hipscat/util.py

```diff
@@ -145,7 +145,123 @@ def gc_dist(lon1, lat1, lon2, lat2):
     lon1 = np.radians(lon1); lat1 = np.radians(lat1)
     lon2 = np.radians(lon2); lat2 = np.radians(lat2)
 
-    return np.degrees(2*np.arcsin(np.sqrt( (np.sin((lat1-lat2)*0.5))**2 + np.cos(lat1)*np.cos(lat2)*(np.sin((lon1-lon2)*0.5))**2 )));
+    return np.degrees(2*np.arcsin(np.sqrt( (np.sin((lat1-lat2)*0.5))**2 + np.cos(lat1)*np.cos(lat2)*(np.sin((lon1-lon2)*0.5))**2 )))
+
+
+def which_cull_and_pixorder(c1, c2):
+    
```

Comment:
Please add some docstrings for utility methods.

Reply:
added more docstrings in the util functions